addons/mail/static/src/components/message/message.js

```diff
@@ -104,6 +104,10 @@ class Message extends Component {
     _constructor() {}
 
     mounted() {
+        // remove all readmore before if any before reinsert them with _insertReadMoreLess
+        for (const el of [...this._contentRef.el.querySelectorAll(':scope .o_Message_readMore')]) {
+            el.remove();
+        }
```

Comment:
The problem comes from read more/less being handle by directly mutating the DOM, therefore these parts of DOM are not in sync with OWL.

Ideally should rewrite this without DOM mutation, but it's hard and better postpone for later...

I'm wondering if this fixup shouldn't be instead moved to `willUnmount`? Maybe not worth it, let's just plan improving that with task-2308951 (could refer to this task in comment as FIXME).

Reply:
Cannot be in `willUnmount` as `_contentRef` is already unmounted

Reply:
That sucks... This makes no sense, but let's not bother with it.